Yoast/Yoast-theme-public
# Stories/1255 remove bottom menu

…igation)

fixes https://github.com/Yoast/yoast.com/issues/1255
Goes along with:  https://github.com/Yoast/yoast.com-ms/pull/111

**How to test**
Check every type of page and check if the navigation is gone. Also, check if the footer and the element above look good together. So no weird arrow's, whitespaces or anything. if you think something went wrong. please note the URL so i can check it.
